src/js/stores/search.js

```diff
@@ -271,36 +281,41 @@ define(function (require, exports, module) {
             var searchTypeInfo = this._registeredSearchTypes[type];
             if (searchTypeInfo) {
                 var items = searchTypeInfo.getOptions();
-                
-                // Get shortest unique paths
-                var ancestors = collection.pluck(items, "pathInfo"),
-                    shortenedPaths = searchTypeInfo.shortenPaths ?
-                        pathUtil.getShortestUniquePaths(ancestors).toJS() : ancestors.toJS();
-
-                var itemMap = items.map(function (item, index) {
-                    var newPathInfo = shortenedPaths[index] || "",
-                        name = item.name,
-                        style;
-                    // Don't show the path info if it is just the same as the item name 
-                    if (name === newPathInfo) {
-                        newPathInfo = "";
-                    }
-
-                    if (name === "") {
-                        name = "Untitled";
-                        style = { "fontStyle": "italic" };
-                    }
-
-                    return {
-                        id: type + "-" + item.id.toString(),
-                        title: name,
-                        pathInfo: newPathInfo,
-                        svgType: item.iconID,
-                        category: item.category,
-                        style: style,
-                        type: "item"
-                    };
-                }.bind(this));
+                var itemMap;
+
+                if (items === null) {
+                    itemMap = [];
+                } else {
+                    // Get shortest unique paths
+                    var ancestors = collection.pluck(items, "pathInfo"),
+                        shortenedPaths = searchTypeInfo.shortenPaths ?
+                            pathUtil.getShortestUniquePaths(ancestors).toJS() : ancestors.toJS();
+
+                    itemMap = items.map(function (item, index) {
+                        var newPathInfo = shortenedPaths[index] || "",
+                            name = item.name,
+                            style;
+                        // Don't show the path info if it is just the same as the item name 
+                        if (name === newPathInfo) {
+                            newPathInfo = "";
+                        }
+
+                        if (name === "") {
+                            name = "Untitled";
```

Comment:
I realize this isn't your code, but it sure seems weird that this string isn't localized. You might just want to file a separate bug for this?